.travis.yml

```diff
@@ -57,6 +59,10 @@ after_success:
       docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD";
       docker push $DOCKER_IMAGE:$DOCKER_TAG;
     fi
+  - if [ "$DOCKER_TAG" == "master" ]; then
```

Comment:
Oh right I forgot about latest, I think we should just stop using it because it often isn't what is says it is. But in the meantime let's do the double push.